frontend/.eslintrc.json

```diff
@@ -0,0 +1,22 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "plugin:prettier/recommended"
+    ],
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
+        "react/react-in-jsx-scope": "off",
+        "semi": [2, "never"]
```

Comment:
1. What does this value mean? I would expect just `"never"` (or to be precise `"always"`) but I can't find in docs what's the correct form for this rule.
2. We are not doing that, and I will tell you why:
```javascript
var globalCounter = { }

(function () {
    var n = 0
    globalCounter.increment = function () {
        return ++n
    }
})()
```
ESLint and guidelines are here to avoid ambiguity in code and disabling semicolons introduces a lot of ambiguities.

Reply:
as settled on meeting semicolns are turned on